types/prettier/index.d.ts

```diff
@@ -374,7 +374,7 @@ export interface Parser<T = any> {
 }
 
 export interface Printer<T = any> {
-    print(path: AstPath<T>, options: ParserOptions<T>, print: (path: AstPath<T>) => Doc): Doc;
+    print(path: AstPath<T>, options: ParserOptions<T>, print: (path: AstPath<T>) => Doc, args: any): Doc;
```

Comment:
Should `unknown` be preferred?

Reply:
The important part is to define the correct number of parameters

Reply:
I mean `args:unknown`